extensions-contrib/orc-extensions/pom.xml

```diff
@@ -44,6 +44,16 @@
             <groupId>org.apache.hive</groupId>
             <artifactId>hive-exec</artifactId>
             <version>${hive.version}</version>
+            <exclusions>
```

Comment:
it will be nice if we can just depend on https://orc.apache.org/news/2018/09/25/ORC-1.5.3/

Reply:
probably yes, but that is outside the scope of this pr